### DISCUSSION TITLE
Transparency: Access various system tids via Advanced Search

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#5516] [Ideas]
*"So, this viewtemplate should sit above the text field... Simple, I just `list-before: ...` eeeh, what's the name of the text field's viewtemplate again? And how do I find it again?..."*

For better transparency, I would like for the **Advanced Search to have another tab** for direct access to some systemic tids. The tab could be named "System" with **sub tabs** containing *ready made lists* for:

* ViewTemplates
* EditViewTemplates
* Styleheets
* Probably also plugins (with accordions to see their shadows). It feels wrong to search for shadows and overwrites via the plugin library.
* The premade searches linked to in Ctrlpanel>Info>Basics could also be here (why should I go to the Ctrlpanel to search these?)
* Icons (ideally an icons gallery)
* Toolbar buttons (in Ctrlpanel>Appearance>Toolbars ... there are no *links* to the buttons so it is tricky to find "templates" for when creating your own buttons - not an uncommon thing to do).

... probably many more?

If needed, there could be a a search field in the subtab for refining the search further (...all stylesheets containing `.foo {`, etc.)

What do you think? I think we have, over the years, have accustomed ourselves to the many hoops we jump through to locate systemic tiddlers but this can easily be simplified.

**Comment by pmario** [comment]
Hi Mat, 
Creating buttons is simple: see: https://github.com/Jermolene/TiddlyWiki5/pull/5294 ... Just vote for it there and play with the wiki that's linked at the PR
